### ISSUE TITLE
Error when loading ckpt for Dreambooth

### ISSUE THREAD

**Issue by Omenizer** [TheLastBen/fast-stable-diffusion#901]
![image](https://user-images.githubusercontent.com/119709270/206148144-75d370ec-6341-431d-9601-9a0965dd1ffc.png)

It's a merged model that I downloaded as safetensors. Because I would get a similar error when trying to load it into Dreambooth, I merged it with 0 weight and saved as checkpoint (poor man's conversion) but it still won't load. 

It will load and work just fine in SD, but not on DB. Anything I can do to train it? Thanks!

**Comment by TheLastBen** [comment]
in DB it converts it to diffusers, the conversion script doesn't support safetensors format

**Comment by Omenizer** [comment]
So it's still safetensors even if it's a ckpt file?

**Comment by TheLastBen** [comment]
> So it's still safetensors even if it's a ckpt file?

Probably, I haven't tested it yet, I will later.

**Comment by Omenizer** [comment]
Thanks!

**Comment by SAC020** [comment]
Getting the same error when trying to load a ckpt trained based on SD20 720px

**Comment by TheLastBen** [comment]
> Getting the same error when trying to load a ckpt trained based on SD20 720px

because it wasn't converted correctly

**Comment by SAC020** [comment]
> > Getting the same error when trying to load a ckpt trained based on SD20 720px
> 
> because it wasn't converted correctly

It was trained with this repo and works in SD. No idea what I did incorrectly, but it was trained during the period of intensive changes following the release of SD20, so it's probably due to that.

**Comment by TheLastBen** [comment]
yes, before the recent changes, the converting script was not working correctly, you will even notice the model doesn't really reflect the trained subject, but now it's fixed, any model converted before the change will not work.

**Comment by SAC020** [comment]
Ok, thank you

**Comment by Omenizer** [comment]
I'm still having this issue, is there any way I can convert these models for use in dreambooth?

I can't use any of the the newer models, none will work.. Only old stuff and Base 1.5
